src/main/resources/swagger.api/parameters/sort-by-location-name-param.yaml

```diff
@@ -0,0 +1,6 @@
+in: query
+name: sortBy
+description: Defines a field to sort by location name.
```

Comment:
Would be nice to have possible values in a description as we have for item/holdings.

Reply:
Task also says that default value should be location name